### DISCUSSION TITLE
Tornare al firmware originale Sonoff Dual R3

### DISCUSSION THREAD

**Discussion by DarkPako84** [indomus/forum#2898] [Hardware & Firmware]
Ciao a tutti, qualche tempo fa acquistai diversi Sonoff Dual R3, di cui 1 l'ho portato subito a Tasmota.
Dopo diversi mesi ho iniziato ad installare i dispositivi per comandare delle tapparelle e mi sto trovando bene con il firmware originale. Ora mi manca da montare l'ultimo Dual R3 ma vorrei tornare al firmware originale, di cui ho il .bin originale.

Il problema è che non riesco più a scrivere il firmware tramite sia un Arduino e sia un Esp8266. Ho provato sia con esptool, tasmotizer, esphome. Mi apre correttamente la porta ma non comunica nulla...
Ho anche ricontrollato diverse volte i collegamenti e mi sembrano a posto. Molto spesso però vedo che in fase di scrittura del firmware la lucetta che ho sul mio arduino, riferito al TX, si accende in maniera molto fioca... cosa potrebbe essere?

**Comment by dante70** [comment]
ciao,non entro nel merito di portare al firmware originale in quanto non lo ho mai fatto.Io ho da qualche mese il problema che non riesco piu a fare flash dei dispositivi esp66(sonoffmini) e esp32(dual r3+mini r4).....credo sia lo stesso problema tuo.fino a ll'estate ho sempre fatto con esphome flash deidispositivi menzionati,ora non riesco.Ho anche aperto una discussione qui ma senza risposta

**Reply by dante70** [reply]
https://github.com/indomus/forum/discussions/2637
